### ISSUE TITLE
Problem with asyncData on page refresh and nuxt generate?

### ISSUE THREAD

**Issue by jesperlandberg** [nuxt/nuxt.js#2111]
I just switched from an API, to having my json files in the static folder. Works fine, except for when i do a page refresh. I then get a 404 and axios createError. I'm also getting these errors when trying to generate the site

```
  async asyncData () {
    let { data } = await axios.get('/data/projects.json')
    return { content: data }
  }
```

Any idea what might be wrong?

**Comment by uptownhr** [comment]
@jesperlandberg what is your base url set to for axios?

**Comment by husayt** [comment]
This seems to be resolved in 1.4.0. Closing. Reopen if still happens.
